scripts/annotate.py

```diff
@@ -13,24 +13,27 @@ def annotate_cells(aggregated_data, barcodes_path):
         path pointing to aggregated dataset
     platemaps : str
         path pointing to platemaps
-
+    output: str
+        name of generated annotated profile
+    configs:
+        path to configuration file
     """
+
     # loading in paramters
-    annotate_ep = Path(snakemake.params["annotate_config"])
-    annotate_config_path = annotate_ep.absolute()
+    annotate_path_obj = Path(configs)
+    annotate_config_path = annotate_path_obj.absolute()
     with open(annotate_config_path, "r") as yaml_contents:
         annotate_configs = yaml.safe_load(yaml_contents)["annotate_configs"]["params"]
 
     # input paths retrived by snakemake
-    annotated_outfile = str(snakemake.output)
     platemap_df = pd.read_csv(barcodes_path)
 
     # annotating the aggregated profiles
     annotate(
         profiles=aggregated_data,
         platemap=platemap_df,
         join_on=annotate_configs["join_on"],
-        output_file=annotated_outfile,
+        output_file=output,
```

Comment:
lets keep variable names consistent. In `aggregate.py` this variable was named `aggregate_file_out`. Let's call this `annotate_file_out`